src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -213,6 +213,7 @@ HARDWARE_INTRINSIC(Sve,           TransposeEven,
 HARDWARE_INTRINSIC(Sve,           TransposeOdd,                                                     -1,      2,      true,  {INS_sve_trn2,       INS_sve_trn2,       INS_sve_trn2,       INS_sve_trn2,       INS_sve_trn2,       INS_sve_trn2,       INS_sve_trn2,       INS_sve_trn2,       INS_sve_trn2,       INS_sve_trn2},    HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_SpecialCodeGen)
 HARDWARE_INTRINSIC(Sve,           UnzipEven,                                                        -1,      2,      true,  {INS_sve_uzp1,       INS_sve_uzp1,       INS_sve_uzp1,       INS_sve_uzp1,       INS_sve_uzp1,       INS_sve_uzp1,       INS_sve_uzp1,       INS_sve_uzp1,       INS_sve_uzp1,       INS_sve_uzp1},    HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_SpecialCodeGen)
 HARDWARE_INTRINSIC(Sve,           UnzipOdd,                                                         -1,      2,      true,  {INS_sve_uzp2,       INS_sve_uzp2,       INS_sve_uzp2,       INS_sve_uzp2,       INS_sve_uzp2,       INS_sve_uzp2,       INS_sve_uzp2,       INS_sve_uzp2,       INS_sve_uzp2,       INS_sve_uzp2},    HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_SpecialCodeGen)
+HARDWARE_INTRINSIC(Sve,           VectorTableLookup,                                                -1,      2,      true,  {INS_sve_tbl,        INS_sve_tbl,        INS_sve_tbl,        INS_sve_tbl,        INS_sve_tbl,        INS_sve_tbl,        INS_sve_tbl,        INS_sve_tbl,        INS_sve_tbl,        INS_sve_tbl},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_BaseTypeFromSecondArg)
```

Comment:
Shouldn't this be `HW_Flag_BaseTypeFromFirstArg` or do we even need this flag?

Reply:
You're right, we don't even need a flag. I initially used the second arg to determine the base jit type as VectorTableLookup has args with mixed data types but of the same size. However, it's not necessary so removed it.